.github/workflows/ci.yml

```diff
@@ -7,24 +7,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        ruby-version:
-          - '2.7'
-          - '3.0'
-          - '3.1'
-          - '3.2'
-        rails-version:
-          - '6.0'
-          - '6.1'
-          - '7.0'
-    name: 'Ruby ${{ matrix.ruby-version }}, Rails ${{ matrix.rails-version }}'
+        # To keep matrix size down, only test highest and lowest rubies.
+        ruby: ["2.7", "3.3"]
+        rails: ["6.1", "7.0", "7.1", "7.2"]
```

Comment:
I’ve removed support for Rails 6.0 as tests are failing on this version. Although the test build was showing as green, it was actually running on Rails 6.1 instead of 6.0. This happened because the Gemfile used `~> 6.0`, which incorrectly installed Rails 6.1. To truly use Rails 6.0, the Gemfile should have specified `~> 6.0.0`.

I chose not to reintroduce [these](https://github.com/bogdan/datagrid/commit/c06bb11f7d4ed754508aa1ed14e6c2b0d266b630) [hacks](https://github.com/bogdan/datagrid/commit/98ae8702999dc25b9babb68eff84819966ccf667), and instead, I believe it’s time to officially drop support for this long end-of-life Rails version.

Here’s a screenshot for reference:

<img width="674" alt="2024-09-04 20-28-18" src="https://github.com/user-attachments/assets/d9bfe00e-9cf9-444f-8316-0f1b70df0344">